### PR TITLE
Us#100 post pulse

### PR DESCRIPTION
**Pull request by mrgutie8**

### CODE REVIEW

File: lib/metabase/endpoint/pulse.rb [hunk 1]
```diff
@@ -11,6 +11,15 @@ module Pulse
       def pulses(**params)
         get('/api/pulse', **params)
       end
+
+      # Create a new Pulse.
+      #
+      # @param params [Hash] Query string
+      # @return [Array<Hash>] Parsed response JSON
+      # @see https://github.com/metabase/metabase/blob/master/docs/api-documentation.md#get-apipulse
```
Comment (grustia01) [inline lib/metabase/endpoint/pulse.rb]: create pulse document url is https://github.com/metabase/metabase/blob/master/docs/api/pulse.md#post-apipulse

File: spec/webmock/pulse_spec.rb [hunk 2]
```diff
@@ -88,7 +88,12 @@
 
     end
 
-  end
+    it 'Creates pulse' do
+      stub_request(:post, "http://localhost:3030/api/pulse")
+        .to_return(status: 200, body: "OK")
 
+      expect(client.create_pulse).to have_requested(:post, "#{host}/api/pulse")
```
Comment (grustia01) [inline spec/webmock/pulse_spec.rb]: https://github.com/metabase/metabase/blob/master/docs/api/pulse.md#post-apipulse mentioned request parameters for create pulse for example name, card, channel
test create pulse with correct request and response params

**Review by apassett — APPROVED** [review]
Igtm

**Review by grustia01 — COMMENTED** [review]
good to merge

**Pull request merged**